pkg/utils/perm_windows.go

```diff
@@ -30,16 +31,38 @@ type FileOwner struct {
 // Note: Very unlikely for this to panic if this function is unable to get the current
 // user. Not being able to get the current user, is a critical problem and nothing
 // can continue so a panic is appropriate.
```

Comment:
Looks like we no longer `panic` so this comment should be updated?

Reply:
I have updated this comment.